graphql_compiler/tests/test_input_data.py

```diff
@@ -1565,6 +1586,27 @@ def has_edge_degree_op_filter_with_fold():  # noqa: D103
         type_equivalence_hints=None)
 
 
+def is_null_op_filter_optional_value():  # noqa: D103
```

Comment:
nit: use `missing` rather than `optional` here, since `optional` is an overloaded that also means something else (`@optional`)